src/time_tagger/hardware_properties/service.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+import TimeTagger as TT
+
+
+@dataclass
+class SetTriggerLevelDto:
+    time_tagger_network_proxy: object
+    channels_voltage: dict[int, float]
+
+
+class TimeTaggerHardwarePropertiesService:
```

Comment:
This is not a service, this should be a dao, as it hold implementation details of the data provider